docs/references/connections.md

```diff
@@ -0,0 +1,233 @@
+# Connections
+* * *
+
+### What are Connections?
+A **connection** refers to the link between an Adapter and other external systems, such as a vCenter Server, other
+databases, or cloud services. These connections allow an Adapter to gather performance and capacity data about your
+virtual or physical infrastructure operations, which can then be analyzed and presented in a unified, comprehensive view. Aria
+Operations uses this data to help automate and simplify operations management.
+
+Creating a connection usually involves specifying the IP address or hostname of the external system, along with the appropriate
+[credentials]() to access that system. After establishing the connection, an Adapter can collect data from the connected system.
+
```

Comment:
We might want to discuss suite-api connection info here as well, since that's a part of a connection. We discuss it a bit in the project config page, but that's more of a 'how are they stored/used', whereas here I think we'd want more of a 'what are they/when do we need them'.